src/main/java/bisq/price/spot/ExchangeRateService.java

```diff
@@ -47,11 +48,22 @@ class ExchangeRateService {
      * @param providers all {@link ExchangeRateProvider} implementations in ascending
      *                  order of precedence
      */
+    @Autowired
```

Comment:
Is that needed? I am not familiar with Spring, but as it was not there before I assume its not needed, same for the added constructors.

Reply:
It was needed with the multiple constructor approach to fix the compiling issue, however I think the solution given in #14  is more accurate and this will be gone when I rebase ;) 
Thanks!